### DISCUSSION TITLE
How to organize async dev & design work?

### DISCUSSION THREAD

**Discussion by paulcretu** [orcasound/orcasite#529] [General]
This discussion has been an ongoing action item in the check-in calls. Moving it here since it's a larger topic.

Goal (as formulated by @scottveirs): devise asynchronous method of prioritizing dev and design team work (ideally including milestones, or other scheduling constraints or goals)

Questions:
- How to break down silos and connect teams better across Orcasound community? (And make sure that there are sufficient volunteers -- devs, UX, PMs…)
- How (often) should UX ask for dev feedback? (as opposed to devs searching through comments for where they are tagged)
- How often should PMs re-poll devs/designer/UX for optimal synchronization calls?
- How often should synchronous standups happen for orcasite (ideally with one person from each team -- dev and design/UX)?

**Comment by Filadora** [comment]
I'm proposing selecting one or a few tasks during each stand-up for the team (both Devs and UX) to work on collaboratively over the next two weeks. I suggest these tasks be based on the project's current needs, with a focus on prioritizing the scientific aspect and the primary goal of preserving the orcas.

**Comment by Filadora** [comment]
I was trying to put together onboarding instructions for the UX Orcasite team members. They include the suggestions I mentioned above. Please have a look and share your feedback.

**Orcasite UX Designer Onboarding Instructions**

Welcome to the UX-Orcasite team! We're happy to have you on board as a volunteer UX designer. This document will guide you through our workflow and help you get started with contributing effectively.

**Overview**

Our design process is closely integrated with the development workflow. Each design task is associated with a GitHub issue. GitHub issues and being created by devs, stakeholders UX design leads, and managers based on the design needs of the project uncovered by various ways including but not limited to user feedback, constraints in developers' work, and stakeholders’ requests. There is a separate Figma page for each task with a GitHub issue number, which helps us track progress and ensure seamless communication between designers and developers. If there is no GitHub number that means that is an issue or suggestion that is coming from user testing results and needs more work before addressing it to the devs.

**Getting Started**

Join Our Communication Channels:
Make sure you have access to our Slack channels and GitHub repository. These are our primary communication and collaboration tools.
Set Up Your Tools:
Ensure you have access to Figma. If you don't have access yet, please contact Darya Sedyh for an invite.

**Workflow**

**Understanding GitHub Issues:**

Each design page in Figma corresponds to a GitHub issue. These issues may be created by developers, Scott (stakeholder), or UX Design leads and Managers based on user research results, dev’s needs or stakeholder’s requests.
If a Figma page does not have an associated GitHub issue number, it indicates that the issue or suggestion is coming from user testing results and requires further refinement before being addressed by the development team.

**Working on Design Tasks:**
When you start working on a design, first check the associated GitHub issue to understand the requirements and context.
If there is no GitHub issue linked to the Figma page, review the user testing results and identify what needs to be improved or clarified. Document your findings and suggestions on the corresponding Figma page and present them to the team during the meeting or by commenting and tagging team members in Figma.
Communicate any uncertainties or questions with the team through our communication channels.

**Submitting Designs:**
Once your design is ready, update the corresponding Figma page and tag the team members and devs for review.
Post a summary of your design updates and the link to the Figma page in the GitHub issue comments after the design approval.

**Iterating Based on Feedback:**
Be open to feedback and ready to iterate on your designs. Collaboration is key to our success.

**Additional Notes**

Documentation: Always keep detailed notes on your design decisions and user research insights. This documentation will help others understand your thought process and maintain consistency across the project.
Collaboration: We value teamwork and open communication. Don't hesitate to reach out if you need help or want to brainstorm ideas.

Thank you for joining us! Your contributions are invaluable, and we're excited to see the impact you'll make!

Here is the link:
https://www.figma.com/design/41JuvNkXJhTUob8HMsJiNC/Orcasite%3A-design-updates?node-id=1-3&t=8uX0kXPvQc21uOMy-1

**Comment by dthaler** [comment]
Using a slack channel plus standups every other week seems fine to me.

**Comment by adrmac** [comment]
Is this project board still current? 
https://github.com/orgs/orcasound/projects/38/views/1
I had some experience organizing async design/dev work at my last company, we typically used weekly scrum meetings to review and prioritize tickets. As we perform UX research, we tend to collect a large backlog of feature ideas that should be discussed and evaluated in terms of effort vs. impact.

**Reply by paulcretu** [reply]
No, it's not up to date, but I'd love to do a cleanup and get it current. And I'm completely in favor of estimating effort vs impact as a way to prioritize the backlog. The way it's set up right now didn't work too well for our workflow, it could use a redo (we could make a new board?)
